pkgs/top-level/all-packages.nix

```diff
@@ -529,6 +529,8 @@ with pkgs;
 
   coordgenlibs  = callPackage ../development/libraries/coordgenlibs { };
 
+  cope = callPackage ../tools/misc/cope { };
+
```

Comment:
```suggestion
```

See overall comment

Reply:
that line was removed and default.nix was moved to pkgs/by-name/co/cope/package.nix